src/main/java/frc/robot/Util.java

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+//
+package frc.robot;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/** Add your docs here. */
+public class Util {
+
+    public static Translation3d getNormalTranslation(Translation3d vec){
+        return(new Translation3d(vec.getX()/vec.getNorm(), vec.getY()/vec.getNorm(), vec.getZ()/vec.getNorm()));
+    }
+
+    public static class PIDConstants{
+        public double mKP;
+        public double mKI;
+        public double mKD;
+        public double mKV;
+        public double mKS;
+        public double mContinuousInputMax;
+        public double mContinuousInputMin;
+        public boolean mIsContinuousInput;
+
+        public PIDConstants(double kP, double kI, double kD, double kS, double kV, double continuousInputMax, double continuousInputMin){
+            mKP = kP;
+            mKI = kI;
+            mKD = kD;
+            mKV = kV;
+            mKS = kS;
+            mContinuousInputMax = continuousInputMax;
+            mContinuousInputMin = continuousInputMin;
+            mIsContinuousInput = true;
+        }
+
+        public PIDConstants(double kP, double kI, double kD, double kS, double kV){
+            this(kP, kI, kD, kS, kV,0,0);
+            mIsContinuousInput = false;
+        }
+
+        public PIDConstants(double kP, double kI, double kD){
+            this(kP, kI, kD, 0, 0,0,0);
+            mIsContinuousInput = false;
+        }
+   
+
+        public PIDController toWPIController(){
+            PIDController pid = new PIDController(mKP, mKI, mKD);
+            if(mIsContinuousInput){pid.enableContinuousInput(mContinuousInputMin, mContinuousInputMax);}
```

Comment:
Generally don't squish if statements on one line like that, expand for clarity.

Reply:
Sorry about that. We fixed it now.